panel/reactive.py

```diff
@@ -529,7 +533,7 @@ def controls(self, parameters=[], jslink=True, **kwargs):
             return controls.layout[0]
         return style.layout[0]
 
-    def jscallback(self, args={}, **callbacks):
+    def jscallback(self, args: Dict={}, **callbacks):
```

Comment:
The `args` type annotation should be `Dict[str, Something]` where I don't know what `Something` is. So I did not add it.